app/design/adminhtml/Magento/backend/Magento_Sales/web/css/source/module/order/_order-account.less

```diff
@@ -27,3 +27,18 @@
         width: 50%;
     }
 }
+.page-create-order {
+    .order-details {
+        &:not(.order-details-existing-customer) {
+            .order-account-information {
+                .field-email {
+                    margin-left: -30px;
```

Comment:
In this case, I guess, we can adjust the old fix on **line 25** by changing it from `margin: 0;` to `margin-top: 0;`, in order to avoid overriding the rules many times.
https://github.com/magento/magento2/pull/21009/files#diff-6a9aa8e01731664126ff5fb4fbfc3198R25

Reply:
If I need to apply css only for non existing customer order, then how we will add in line number 25, I think that will reflect for both cases existing customer order and non existing customer order.

Reply:
Good point, thank you.